mage_ai/cli/main.py

```diff
@@ -26,92 +26,81 @@ def list_commands(self, ctx: Context):
 
 # Defaults
 
-INIT_PROJECT_PATH_DEFAULT = typer.Argument(..., help='path of the Mage project to be created.')
+INIT_PROJECT_PATH_DEFAULT = typer.Argument(
+    ..., help="path of the Mage project to be created."
```

Comment:
nit: we actually prefer using single quote in the code.

Reply:
Sorry, my auto formatter on save :( - I can revert